com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -724,6 +727,30 @@ private void ExtractNetworkMetricsFromPipeline(NetworkPipeline pipeline, Network
                 networkMetricsContext->PacketReceivedCount = 0;
             }
         }
+
+        private void ExtractRtt(NetworkConnection networkConnection, ulong transportClientId)
```

Comment:
The `transportClientId` argument is unused (it's also technically redundant with the network connection).

Reply:
Also, might be nice to have this method return the RTT value instead (and have its caller send it to `NetworkMetrics`). This way the method could easily be reused for other purposes (like implementing `GetCurrentRtt`).

Not needed in the context of this PR, though.

Reply:
^ I agree with this, there's been several callouts recently for needing RTT so it's worth making the RTT logic atomic so it can be reused